app.js

```diff
@@ -53,8 +53,9 @@ function addClass(insertedDiv, letter, i) {
 function gameSetup() {
   addLetterListeners();
   insertNextImg();
-  let rndPickedProverb = proverbs[getRandomInt(proverbs.length)];
-  sentence.innerHTML = encryptSentence(rndPickedProverb);
+  state.proverb = proverbs[getRandomInt(proverbs.length)].toLowerCase();
+  state.displayedProverb = encryptSentence(state.proverb);
+  sentence.innerHTML = state.displayedProverb;
```

Comment:
I think that you should skip the `displayProverb` in the state. It's name clearly says that it is some kind of display. You have all data already in provereb, `encryptSentece` is only the version of how it should like. 

So why not just:
`sentence.innerHtml = encryptSentence(state.proverb)`